scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -325,7 +333,12 @@ task ExtractFilterTask {
       ~{"--excess-alleles-threshold " + excess_alleles_threshold} \
       -L ~{intervals} \
       --dataset-id ~{dataset_id} \
-      --project-id ~{query_project}
+      --project-id ~{query_project} \
+      --cost-observability-tablename ~{cost_observability_tablename} \
+      --call-set-identifier ~{call_set_identifier} \
+      --wdl-step GvsCreateFilterSet \
+      --wdl-call ExtractFilterTask \
+      --shard-identifier ~{intervals_name}
   >>>
```

Comment:
fully out of the scope of this pr, but we have so many params! I wonder if we could cut down on a lot of the fq ones--like would we ever not know what the alt-allele-table was called if we have the proj and dataset?